src/main.cpp

```diff
@@ -0,0 +1,28 @@
+al#include <Arduino.h>
+#include "mcp23017.h"
+
+// TODO: declare mcp23017 object
+Mcp23017 mcp23017(0x20);
+
+
+void setup() {
+  // TODO: initialize i2c and mcp23017 object
+  Wire.begin();
+  uint8_t directions[8] = {1, 1, 1, 1, 1, 1, 1, 0};
```

Comment:
Good catch on the array not being 8 numbers long, but you still haven't addressed the problem I mentioned last time. According to the data sheet, 1 = input and 0 = output. If you wanted to turn on an LED with pin 0, would that require an input to the IO expander or an output from the IO expander?